src/components/download/LegendPosition.js

```diff
@@ -10,64 +10,18 @@ export const legendPositions = [
     'bottomright',
 ];
 
-const styles = theme => ({
-    root: {
-        width: 150,
-        paddingLeft: 30,
-    },
-    label: {
-        fontSize: 14,
-        padding: 5,
-    },
-    position: {
-        position: 'relative',
-        display: 'inline-block',
-        backgroundColor: theme.palette.background.default,
-        width: 50,
-        height: 50,
-        margin: '1px 5px',
-        outline: `1px solid ${theme.palette.divider}`,
-        cursor: 'pointer',
-    },
-    legend: {
-        position: 'absolute',
-        width: 12,
-        height: 12,
-        backgroundColor: theme.palette.action.active,
-    },
-    selected: {
-        outline: `3px solid ${theme.palette.primary.main}`,
-    },
-    topleft: {
-        top: 8,
-        left: 8,
-    },
-    topright: {
-        top: 8,
-        right: 8,
-    },
-    bottomleft: {
-        bottom: 8,
-        left: 8,
-    },
-    bottomright: {
-        bottom: 8,
-        right: 8,
-    },
-});
-
-export const LegendPosition = ({ position, onChange, classes }) => (
-    <div className={classes.root}>
-        <div className={classes.label}>{i18n.t('Legend position')}</div>
+export const LegendPosition = ({ position, onChange }) => (
+    <div className={styles.root}>
+        <div className={styles.label}>{i18n.t('Legend position')}</div>
         {legendPositions.map(pos => (
             <div
                 key={pos}
-                className={`${classes.position} ${
-                    pos === position ? classes.selected : ''
+                className={`${styles.position} ${
```

Comment:
Here you can use `cx` for composing the class names, it's a bit cleaner and we used it around.

Reply:
Switched to cx - thanks for the advice!